### DISCUSSION TITLE
Change generated class names

### DISCUSSION THREAD

**Discussion by Ricksters2000** [emotion-js/emotion#2783] [Q&A]
Is it possible to change the generated class names? I thought of doing this with the createCache and CacheProvider but it can only add a prefix, I was wondering if its possible to change more than just a prefix.

**Comment by srmagura** [comment]
Class names are always of the form `cacheKey-hash-label` with the `-label` part being optional. How do you want to customize it and what problem would that solve?

**Reply by Ricksters2000** [reply]
I was thinking if it would be possible to customize the hash since the project I'm working on displays articles between different sites and I would like for the class names to be generated differently between those sites.
